packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -275,6 +284,13 @@ class TimelineScreen extends Screen {
     }
   }
 
+  void _clearTimeline() {
+    framesBarChart.frameUIgraph.reset();
+    flameChart.attribute('hidden', true);
```

Comment:
should we be clearing the flameChart as well?

Reply:
The flame chart will automatically update when the next frame is selected. It will be hidden until then, so this should be fine.